### ISSUE TITLE
Attempting to change "Copy to output directory" after changing the build action from "C# compiler" -> "Content" fails

### ISSUE THREAD

**Issue by davkean** [dotnet/project-system#1316]
1. Select a C# file
2. In Properties, change build action from "C# compiler" -> "Content":
3. Change "Copy to output directory" -> "Always"

Expected: For this to succeed
Actual:
```

An error has occurred while saving the edited properties listed below:
	Copy to Output Directory
One or more values are invalid. Mismatched PageRule with the wrong ItemType.

```

**Comment by kaushalparik27** [comment]
Is there any updates on this issue. I am facing similar one with VS2017 Enterprise Trial when I try to follow similar steps with a .json file.

**Comment by davkean** [comment]
If you change selection to another file and then back again it should work.

**Comment by kaushalparik27** [comment]
@davkean worked! Thx!

**Comment by Pilchie** [comment]
@davkean - any idea on a `Feature` label for this one?

**Comment by davkean** [comment]
This should be now fixed since @jviau started refreshing in this case.

**Comment by Pilchie** [comment]
@jmarolf can you verify and close?

**Comment by davkean** [comment]
Yep, verified this was fixed, I think in 15.3/15.4 timeframe.

**Comment by Pilchie** [comment]
Or, nevermind @jmarolf - thanks @davkean

**Comment by jmarolf** [comment]
:+1:

**Comment by michael-freidgeim-webjet** [comment]
I am using VS Enterprise 2017 (ver 15.7.2) and have the same message when changing properties of html file to "Content" and "Copy if newer" at the same time. But workaround to apply each setting separately works.

**Comment by davkean** [comment]
@michael-freidgeim-webjet Which properties window are you using? It is the floating Properties or the Properties tool window? This is known in floating Properties and doesn't repro in 15.8 because we removed that window.

**Comment by michael-freidgeim-webjet** [comment]
On Solution explorer (for asp.net core solution targeting Net 4.6.1) I right-click and select Properties. 
It is a dialog box with configuration and platform options

**Comment by davkean** [comment]
Yes, thanks. We're aware of that.

**Comment by ericoff** [comment]
I still get this when doing it to multiple files using the Properties Tool Window, FWIW. I also noticed that the "Build Action" changes, but the "Copy To Output Directory" does not. As reported, doing them individually works, but that's a pain.

**Comment by smflorentino** [comment]
@ericoff your workaround worked great, thank you! I am having the same issue, VS Enterprise 15.7.5.

**Comment by smflorentino** [comment]
@Pilchie can you re-open this issue? Or is there another one tracking the issues in 15.7?

**Comment by smflorentino** [comment]
Never mind - seems like it's fixed in 15.8 per #3607
